### ISSUE TITLE
some domains are losing their agency in the a11y scanning process

### ISSUE THREAD

**Issue by gbinal** [18F/pulse#521]
A handful of domains are missing an agency in the a11y data.  Note `app.gov` and `apps.gov` [here](https://staging.pulse.cio.gov/accessibility/domains/#q=app).  

This may likely be addressed by the updated scanning but we should check.

**Comment by gbinal** [comment]
I've confirmed that this is not at issue in the HTTPS and DAP sections so I think it's a one time scanning issue.

**Comment by gbinal** [comment]
Unfortunately, this is still outstanding.  On the [`by domain` section](https://staging.pulse.cio.gov/accessibility/domains/), it can be easily seen by filtering by Agency.  

![screen shot 2016-08-25 at 4 15 02 am](https://cloud.githubusercontent.com/assets/633088/17961572/93c5104c-6a7a-11e6-8f3e-9e009b289338.png)

![screen shot 2016-08-25 at 4 15 14 am](https://cloud.githubusercontent.com/assets/633088/17961577/981bd34c-6a7a-11e6-819d-42f3882a2732.png)

**Comment by gbinal** [comment]
Just to update, the domains still at issue here are: 

````
amberalert.gov
bjs.gov
blmntl.gov
business.gov
crimevictims.gov
fdms.gov
fueleconomy.gov
isotope.gov
isotopes.gov
namus.gov
nccrc.gov
ncrc.gov
nij.gov
scidac.gov
smart.gov
sns.gov
ucrdatatool.gov
````

**Comment by gbinal** [comment]
temporarily addressed by #550

**Comment by gbinal** [comment]
This is fixed in staging and should not recur.  However I'd like to keep this issue open until we've had a chance to re-run the scan and confirm that it doesn't repeat.

**Comment by gbinal** [comment]
Done
